<!DOCTYPE html>
<html lang="pt">

<head>

	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="">
	<meta name="author" content="">
	<link rel="icon" href="img/logo.png">
	<title>Legionella Pneumophila</title>

	<!-- Bootstrap Core CSS -->
	<link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">

	<!-- MetisMenu CSS -->
	<link href="bower_components/metisMenu/dist/metisMenu.min.css" rel="stylesheet">

	<!-- DataTables CSS -->
	<link href="bower_components/datatables-plugins/integration/bootstrap/3/dataTables.bootstrap.css" rel="stylesheet">

	<!-- DataTables Responsive CSS -->
	<link href="bower_components/datatables-responsive/css/dataTables.responsive.css" rel="stylesheet">

	<!-- Custom CSS -->
	<link href="dist/css/sb-admin-2.min.css" rel="stylesheet">

	<!-- Custom Fonts -->
	<link href="bower_components/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">

	<link href="bower_components/timeline.css" rel="stylesheet" type="text/css">

	<!-- DataTables CSS -->
	<link href="bower_components/datatables-plugins/integration/bootstrap/3/dataTables.bootstrap.css" rel="stylesheet">

	<!-- DataTables Responsive CSS -->
	<link href="bower_components/datatables-responsive/css/dataTables.responsive.css" rel="stylesheet">


	<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
	<!--[if lt IE 9]>
		<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
		<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
	<![endif]-->

</head>

<body>

	<div id="wrapper">

		<nav class="navbar navbar-default navbar-static-top" role="navigation" style="margin-bottom: 0">
			<div class="navbar-header">
				<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</button>
				<img class="navbar-brand" src="img/logo.png" alt=" "/>
				<a class="navbar-brand">Legionella Pneumophila</a>
			</div>

			<div class="navbar-default sidebar" role="navigation">
				<div class="sidebar-nav navbar-collapse">
					<ul class="nav" id="side-menu">
						<li>
							<a href="index.html"><i class="fa fa-home fa-fw"></i> Home</a>
						</li>
						<li>
							<a href="#"><i class="fa fa-gear fa-fw"></i> Trabalho realizado<span class="fa arrow"></span></a>
							<ul class="nav nav-second-level">
								<li>
									<a href="literatura.html">Análise de literatura</a>
								</li>
								<li>
									<a href="sequencia.html">Análise da sequência</a>
								</li>
								<li>
									<a href="homologias.html">Análise de homologias por Blast</a>
								</li>
								<li>
									<a href="analise.html">Análise de proteínas</a>
								</li>
								<li>
									<a href="anotacao.html">Anotação funcional</a>
								</li>
								<li>
									<a href="filogenia.html">Alinhamento múltiplo e filogenia</a>
								</li>
								<li>
									<a href="redes.html">Redes metabólica e regulatória</a>
								</li>
								<li>
									<a href="drugs.html">Resistência a drogas e virulência</a>
								</li>
							</ul>
							<!-- /.nav-second-level -->
						</li>
						<li>
							<a href="table.html"><i class="fa fa-table fa-fw"></i> Tabelas resumo</a>
						</li>
						<li>
							<a href="ferramentas.html"><i class="fa fa-wrench fa-fw"></i> Ferramentas utilizadas</a>
						</li>
					</ul>
				</div>
				<!-- /.sidebar-collapse -->
			</div>
			<!-- /.navbar-static-side -->
		</nav>
		<div id="page-wrapper">
			<div class="row">
				<div class="col-lg-12">
					<h2 class="page-header">Relatório de análise genómica</h2>
				</div>
				<!-- /.col-lg-12 -->
			</div>
			<div class="row">
				<div class="col-lg-12">
					<div class="panel panel-default">
						<div class="panel-heading">
							<br/>
						</div>
						<!-- /.panel-heading -->
						<div class="panel-body">
							<div class="col-lg-10 col-lg-offset-1">
<h2>Interações Metabólicas e Regulatórias</h2><br/>
<p><big>Para a construção duma rede metabólica dum organismo é necessário conhecer todas as suas reações e enzimas que as catalisam, os produtos dessas reações e como são transportados, o modo como esses processos são regulados e muitos outros fatores. Assim, a construção de tabelas sobre os genes que codificam e regulam os vários intervenientes desta complexa rede é crucial.</big></p>

<br/><h3>Genes com função metabólica</h3><br/>
<p><big>Através da consulta da base de dados KEGG foi possível registar as vias metabólicas em que estes genes metabólicos intervêm, bem como as reações. Os resultados estão apresentados na seguinte <a target="_blank" href="https://github.com/Hubble83/lb/tree/master/scripts/intMetReg/genesFuncMetabolica.xlsx">Tabela</a>, replicada em seguida.</big></p><br/><br/><br/>

<div class="dataTable_wrapper" style="margin:0; padding:0; display:table">
	<table class="table table-compressed table-bordered" id="dataTables-example1">
		<thead>
			<tr>
<th>GeneID</th><th>Locus tag</th><th>EC Number</th><th>KEGG Orthology</th><th>KEGG reaction's</th><th>KEGG Pathways</th></tr>
			</tr>
		</thead>
	<tbody>
<tr class="odd"><td>19833668</td><td>lpg2102</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833705</td><td>lpg2139</td><td>EC:1.4.3.5</td><td>K00275</td><td>R00277, R00278</td><td>lpn00750 - Vitamin B6 metabolism<br/>lpn01100 - Metabolic pathways<br/>lpn01120 - Microbial metabolism in diverse environments</td></tr>
<tr class="odd"><td>19833666</td><td>lpg2100</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833724</td><td>lpg2158</td><td>EC:2.1.1.17, 2.1.1.71</td><td>K00570</td><td> R02056, R03424</td><td>lpn00564 - Glycerophospholipid metabolism,<br/>lpn01100 - Metabolic pathways,<br/>lpn01110 Biosynthesis of secondary metabolites</td></tr>
<tr class="odd"><td>19833738</td><td>lpg2172</td><td>EC:3.3.1.1</td><td>K01251 </td><td> R04936</td><td>lpn00270 - Cysteine and methionine metabolism<br/>lpn01100 -Metabolic pathways</td></tr>
<tr class="even"><td>19833742</td><td>lpg2176</td><td>EC:4.1.1.-</td><td>K16239</td><td></td><td></td></tr>
<tr class="odd"><td>19833762</td><td>lpg2196</td><td>EC:4.3.1.12</td><td>K01750</td><td>R00671</td><td>Arginine and proline metabolism<br/>lpn01110 - Biosynthesis of secondary metabolites<br/>lpn01130 - Biosynthesis of antibiotics<br/>lpn01230 - Biosynthesis of amino acids</td></tr>
<tr class="even"><td>19833763</td><td>lpg2197</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833774</td><td>lpg2208</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833782</td><td>lpg2216</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833792</td><td>lpg2226</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833793</td><td>lpg2227</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833794</td><td>lpg2228</td><td>EC:2.3.1.180</td><td>K00648</td><td>R10707</td><td>lpn00061 - Fatty acid biosynthesis<br/>lpn01100 - Metabolic pathways<br/>lpn01212 - Fatty acid metabolism</td></tr>
<tr class="even"><td>19833796</td><td>lpg2230</td><td>EC:6.2.1.3</td><td>K01897</td><td>R00390 > R01280</td><td><br/>lpn00071 - Fatty acid degradation<br/>lpn01100- Metabolic pathways<br/>lpn01212 - Fatty acid metabolism<br/>lpn02024 - Quorum sensing</td></tr>
<tr class="odd"><td>19833797</td><td>lpg2231</td><td>EC:1.1.1.100</td><td>K00059</td><td>R02767 > R04533,<br/>R04534, R04536,<br/>R04543, R04566,<br/>R04953, R04964,<br/>R07763</td><td>lpn00061 - Fatty acid biosynthesis<br/>lpn00780 - Biotin metabolism<br/>lpn01040 - Biosynthesis of unsaturated fatty acids<br/>lpn01100 - Metabolic pathways<br/>lpn01212 - Fatty acid metabolism</td></tr>
<tr class="even"><td>19833798</td><td>lpg2232</td><td>EC:2.3.1.180</td><td>K00648</td><td>R10707</td><td>lpn00061 - Fatty acid biosynthesis<br/>lpn01100 - Metabolic pathways<br/>lpn01212 - Fatty acid metabolism</td></tr>
<tr class="odd"><td>19833801</td><td>lpg2235</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833806</td><td>lpg2241</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833808</td><td>lpg2243</td><td>EC:2.4.2.9</td><td>K00761</td><td>R00966</td><td>lpn00240 - Pyrimidine metabolism<br/>lpn01100 - Metabolic pathways</td></tr>
<tr class="even"><td>19833814</td><td>lpg2249</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833815</td><td>lpg2250</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="even"><td>19833816</td><td>lpg2251</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833817</td><td>lpg2252</td><td>EC:6.3.1.2</td><td>K01915</td><td>R00253</td><td>lpn00220 - Arginine biosynthesis<br/>lpn00250 - Alanine, aspartate and glutamate metabolism<br/>lpn00630 - Glyoxylate and decarboxylate metabolism<br/>lpn00910 - Nitrogen metabolism<br/>lpn01100 - Metabolism pathways<br/>lpn01120 - Microbial metabolism in diverse environments<br/>lpn01230 - Biosynthesis of amino acids<br/>lpn02020 - Two-component system</td></tr>
<tr class="even"><td>19833826</td><td>lpg2261</td><td>EC:2.3.1.8</td><td>K00625</td><td>R00230</td><td>lpn00430 - Taurine and hypotaurine metabolism<br/>lpn00620 - Pyruvate metabolism<br/>lpn00640 - Propanoate metabolism<br/>lpn00680 - Methane metabolism<br/>lpn01100 - Metabolic pathways<br/>lpn01120 - Microbial metabolism in diverse environments<br/>lpn01200 - Carbon metabolism</td></tr>
<tr class="odd"><td>19833827</td><td>lpg2262</td><td>EC:2.7.2.1</td><td>K00925</td><td>R00315</td><td>lpn00430 - Taurine and hypotaurine metabolism<br/>lpn00620 - Pyruvate metabolism<br/>lpn00640 - Propanoate metabolism<br/>lpn00680 - Methane metabolism<br/>lpn01100 - Metabolic pathways<br/>lpn01120 - Microbial metabolism in diverse environments<br/>lpn01200 - Carbon metabolism</td></tr>
<tr class="even"><td>19833839</td><td>lpg2274</td><td>EC:3.1.4.46</td><td>K01126</td><td>R00857, R01030, R01470</td><td>lpn00564 - Glycerophospholipid metabolism</td></tr>
<tr class="odd"><td>19833841</td><td>lpg2276</td><td>EC:1.4.1.9</td><td>K00263</td><td>R01088</td><td>lpn00280 - Valine, leucine and isoleucine degradation <br/>lpn00290 - Valine, leucine and isoleucine biosynthesis<br/>lpn01100 - Metabolic pathways<br/>lpn01110 - Biosynthesis of secondary metabolites<br/>lpn01130 - Biosynthesis of antibiotics</td></tr>
<tr class="even"><td>19833842</td><td>lpg2277</td><td></td><td>No KO assigned</td><td></td><td></td></tr>
<tr class="odd"><td>19833843</td><td>lpg2278</td><td>EC:1.13.11.27</td><td>K00457</td><td>R02521</td><td>lpn00130 - Ubiquinone and other terpenoid-quinone biosynthesis<br/>lpn00350 - Tyrosine metabolism<br/>lpn00360 - Phenylalanine metabolism<br/>lpn01100 - Metabolic pathways</td></tr>
<tr class="even"><td>19833844</td><td>lpg2279</td><td>EC:3.7.1.2</td><td>K16171</td><td>R01364</td><td>lpn00350 - Tyrosine metabolism<br/>lpn00643 - Styrene degradation<br/>lpn01100 - Metabolic pathways<br/>lpn01120 - Microbial metabolism in diverse environments</td></tr>
<tr class="odd"><td>19833845</td><td>lpg2280</td><td>EC:5.2.1.2</td><td>K01800</td><td>R03181</td><td>lpn00350 - Tyrosine metabolism<br/>lpn00643 - Styrene degradation<br/>lpn01100 - Metabolic pathways<br/>lpn01120 - Microbial metabolism in diverse "</td></tr>
<tr class="even"><td>19833847</td><td>lpg2282</td><td>EC:6.1.1.22</td><td>K01893</td><td>R03648</td><td>lpn00970 - Aminoacyl-tRNA biosynthesis</td></tr>
<tr class="odd"><td>19833852</td><td>lpg2287</td><td>EC:1.7.-.-</td><td>K01118</td><td></td><td></td></tr>
<tr class="even"><td>19833858</td><td>lpg2293</td><td>EC:2.7.8.5</td><td>K00995</td><td>R01801</td><td>lpn00564 - Glycerophospholipid metabolism
lpn01100 - Metabolic pathways</td></tr>
<tr class="odd"><td>19833859</td><td>lpg2294</td><td>EC:3.1.3.18</td><td>K01091</td><td>R01334</td><td>lpn00630 - Glyoxylate and decarboxylate metabolism<br/>lpn01100 - Metabolic pathways<br/>lpn01110 - Biosynthesis of secondary metabolites<br/>lpn01130 - Biosynthesis of antibiotics</td></tr>
<tr class="even"><td>19833862</td><td>lpg2297</td><td>EC:3.1.26.12</td><td>K08300</td><td></td><td>lpn03018 - RNA degradation</td></tr>
<tr class="odd"><td>19833867</td><td>lpg2302</td><td>EC:1.2.1.11</td><td>K00133</td><td>R02291</td><td>lpn00260 - Glycine, serine and threonine metabolism<br/>lpn00261 - Monobactam biosynthesis<br/>lpn00270 - Cysteine and methionine metabolism<br/>lpn00300 - Lysine biosynthesis<br/>lpn01100 - Metabolic pathways<br/>lpn01110 - Biosynthesis of secondary metabolites<br/>lpn01120 - Microbial metabolism in diverse environments<br/>lpn01130 - Biosynthesis of antibiotics<br/>lpn01210 - 2-Oxocarboxylic acid metabolism<br/>lpn01230 - Biosynthesis of amino acids</td></tr>
<tr class="even"><td>19833868</td><td>lpg2303</td><td>EC:4.2.3.5</td><td>K01736</td><td>R01714</td><td>lpn00400 - Phenylalanine, tyrosine and tryptophan biosynthesis<br/>lpn01100 - Metabolic pathways<br/>lpn01110 - Biosynthesis of secondary metabolites<br/>lpn01130 - Biosynthesis of antibiotics<br/>lpn01230 - Biosynthesis of amino acids</td></tr>
<tr class="odd"><td>19833874</td><td>lpg2309</td><td>EC:1.1.1.94</td><td>K00057</td><td>R00842, R00844</td><td>lpn00564 - Glycerophospholipid metabolism<br/>lpn01110 - Biosynthesis of secondary metabolites</td></tr>
<tr class="even"><td>19833875</td><td>lpg2310</td><td>EC:5.1.1.3</td><td>K01776</td><td>R00260</td><td>lpn00471 - D-Glutamine and D-glutamate metabolism<br/>lpn01100 - Metabolic pathways</td></tr>
</tbody>
</table>
</div>

<br/><br/><br/><h3>Genes com função transportadora</h3><br/>
<p><big>Estes genes com função transportadora podem atuar de formas muito diversificadas, codificando um vasto número de diferentes proteínas transportadoras, daí o tipo de transportador ser tão variado. Os produtos transportados também são bastante distintos, podendo variar desde macromoléculas a pequenos iões. Para recolher estas informações usamos a base de dados TransportDB em que os dados são apresentados na seguinte <a target="_blank" href="https://github.com/Hubble83/lb/tree/master/scripts/intMetReg/tabelaTransporte.xlsx">Tabela</a>, replicada em seguida.</big></p><br/><br/>
</div>
<div class="col-lg-8 col-lg-offset-2">


<div class="dataTable_wrapper" style="margin:0; padding:0; display:table">
	<table class="table table-compressed table-bordered" id="dataTables-example2">
		<thead>
			<tr>
				<th>ID</th>
				<th>Locus Tag</th>
				<th>TC Number</th>
				<th>Família</th>
				<th>Classe</th>
				<th>Substrato</th>
			</tr>
		</thead>
		<tbody>
<tr class='odd'><td>19833645</td><td>lpg2079</td><td>3.A.7</td><td>IVSP</td><td>ATP-Dependent</td><td>Unclassified</td></tr>
<tr class='even'><td>19833657</td><td>lpg2091</td><td>3.A.7</td><td>IVSP</td><td>ATP-Dependent</td><td>Unclassified</td></tr>
<tr class='odd'><td>19833689</td><td>lpg2123</td><td>2.A.4</td><td>CDF</td><td>Secondary Transporter</td><td>cation efflux</td></tr>
<tr class='even'><td>19833692</td><td>lpg2126</td><td>2.A.72</td><td>KUP</td><td>Secondary Transporter</td><td>potassium ion uptake</td></tr>
<tr class='odd'><td>19833699</td><td>lpg2133</td><td>2.A.1</td><td>MFS</td><td>Secondary Transporter
</td><td>multidrug efflux</td></tr>
<tr class='even'><td>19833700</td><td>lpg2134</td><td>2.A.6</td><td>RND</td><td>Secondary Transporter
</td><td>cobalt/zinc/cadmium ion efflux (HME subfamily)</td></tr>
<tr class='odd'><td>19833718</td><td>lpg2152</td><td>3.A.1</td><td>ABC</td><td>ATP-Dependent</td><td>multidrug</td></tr>
<tr class='even'><td>19833744</td><td>lpg2178</td><td>2.A.6</td><td>RND</td><td>Secondary Transporter</td><td>multidrug/solvent efflux (HAE1 subfamily)</td></tr>
<tr class='odd'><td>19833755</td><td>lpg2189</td><td>9.A.30</td><td>TerC</td><td>Unclassified</td><td>tellurium ion efflux</td></tr>
<tr class='even'><td>19833759</td><td>lpg2193</td><td>2.A.53</td><td>SulP</td><td>Secondary Transporter</td><td>sulfate</td></tr>
<tr class='odd'><td>19833764</td><td>lpg2198</td><td>2.A.17</td><td>POT</td><td>Secondary Transporter</td><td>proton:dipeptide/tripeptide symporter</td></tr>
<tr class='even'><td>19833800</td><td>lpg2234</td><td>2.A.1</td><td>MFS</td><td>Secondary Transporter</td><td>multidrug efflux</td></tr>
<tr class='odd'><td>19833810</td><td>lpg2245</td><td>2.A.23</td><td>DAACS</td><td>Secondary Transporter</td><td>proton/sodium ion:glutamate/aspartate symporter</td></tr>
<tr class='even'><td>19833838</td><td>lpg2273</td><td>3.A.1</td><td>ABC</td><td>ATP-Dependent</td><td>sugar</td></tr>
<tr class='odd'><td>19833849</td><td>lpg2284</td><td>3.A.1</td><td>ABC</td><td>ATP-Dependent</td><td>lipoprotein</td></tr>
<tr class='even'><td>19833850</td><td>lpg2285</td><td>3.A.1</td><td>ABC</td><td>ATP-Dependent</td><td>lipoprotein releasing</td></tr>
</tbody>
</table>
</div>

</div>
<div class="col-lg-10 col-lg-offset-1">

<br/><h3>Genes com função reguladora</h3><br/>
<p><big>Se não houvesse um mecanismo de regulação por trás deste conjunto de reações e transportadores, o organismo não sobreviveria. Para isso existem genes reguladores, para além de outros fatores, que regulam a concentração de inibidores ou simplesmente a transcrição ou tradução de uma determinada enzima, sendo que esta regulação tanto pode ser positiva como negativa, dependendo das necessidades da célula. Os fatores de transcrição são um exemplo de proteínas que regulam a transcrição ligando-se ao DNA e permitindo que a enzima RNA-Polimerase faça a transcrição. Na seguinte <a target="_blank" href="https://github.com/Hubble83/lb/tree/master/scripts/intMetReg/tabelaReguladora.xlsx">Tabela</a> (replicada em seguida) estão representados os genes que codificam os fatores de transcrição e têm essa função reguladora, onde alguns dados foram obtido na base de dados DBD EMBL-EBI.</big></p><br/><br/>

<div class="dataTable_wrapper" style="margin:0; padding:0; display:table">
	<table class="table table-compressed table-bordered" id="dataTables-example3">
<thead>
	<tr>
		<th>ID</th>
		<th>Locus Tag</th>
		<th>Strand</th>
		<th>Uniprot ID</th>
		<th>Revision Grade</th>
		<th>Protein Acession Number</th>
		<th>Protein Name</th>
		<th>Protein size (AA)</th>
		<th>Protein Class</th>
		<th>Family name</th>
	</tr>
</thead>
<tbody>
<tr class="odd"><td>9833687</td><td>lpg2121</td><td>1</td><td>Q5ZTP0</td><td>1/5</td><td>YP_096134.1</td><td>Cold-shock DNA binding domain protein</td><td>69</td><td>Regulation</td><td>Nucleic acid-binding proteins, ‘Cold-shock’ DNA-binding domain</td></tr>
<tr class="even"><td>9833704</td><td>lpg2138</td><td>1</td><td>Q5ZTM3</td><td>1/5</td><td>YP_096151.1</td><td>LysR family transcriptional regulator</td><td>304</td><td>Regulation</td><td>Bacterial regulatory helix-turn-helix protein, lysR family, LysR substrate binding domain</td></tr>
<tr class="even"><td>9833706</td><td>lpg2140</td><td>-1</td><td>Q5ZTM1</td><td>1/5</td><td>YP_096153.1</td><td>MarR family transcriptional regulator</td><td>126</td><td>Regulation</td><td>HxlR-like helix-turn-helix</td></tr>
<tr class="even"><td>9833729</td><td>lpg2163</td><td>-1</td><td>Q5ZTJ8</td><td>1/5</td><td>YP_096176.1</td><td>AraC family transcriptional regulator</td><td>281</td><td>Regulation</td><td>Homeodomain-like, Probable bacterial effector-binding domain</td></tr>
<tr class="even"><td>9833733</td><td>lpg2167</td><td>-1</td><td>Q5ZTJ4</td><td>1/5</td><td>YP_096180.1</td><td>DeoR family transcriptional regulator</td><td>234</td><td>Regulation</td><td>“Winged helix” DNA-binding domain</td></tr>
<tr class="even"><td>9833828</td><td>lpg2264</td><td>1</td><td>Q5ZT98</td><td>1/5</td><td>YP_096276.1</td><td>Hypothetical protein</td><td>104</td><td>Regulation</td><td></td></tr>
<tr class="even"><td>9833851</td><td>lpg2288</td><td>1</td><td>Q5ZT74</td><td>1/5</td><td>YP_096300.1</td><td>LysR family transcriptional regulator</td><td>306</td><td>Regulation</td><td>“Winged helix” DNA-binding domain, Periplasmic binding protein-like II</td></tr>
</tbody>
</table>
</div><br/><br/>


							</div>
						</div>
					</div>
				</div>
				<!-- /.col-lg-8 -->

			</div>
		</div>
		<!-- /#page-wrapper -->

	</div>
	<!-- /#wrapper -->

	<!-- jQuery -->
	<script src="bower_components/jquery/dist/jquery.min.js"></script>

	<!-- Bootstrap Core JavaScript -->
	<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>

	<!-- Metis Menu Plugin JavaScript -->
	<script src="bower_components/metisMenu/dist/metisMenu.min.js"></script>

	<!-- Custom Theme JavaScript -->
	<script src="dist/js/sb-admin-2.min.js"></script>

	<!-- DataTables JavaScript -->
	<script src="bower_components/datatables/media/js/jquery.dataTables.min.js"></script>
	<script src="bower_components/datatables-plugins/integration/bootstrap/3/dataTables.bootstrap.min.js"></script>

	<script>
	$(document).ready(function () {
		$('#dataTables-example1').DataTable({
				responsive: true
		});
		$('#dataTables-example2').DataTable({
				responsive: true
		});
		$('#dataTables-example3').DataTable({
				responsive: true
		});
	});
	</script>

</body>

</html>
